integration/rpctest/node.go

```diff
@@ -115,10 +115,6 @@ func (n *nodeConfig) arguments() []string {
 	args = append(args, fmt.Sprintf("--rpccert=%s", n.certFile))
 	// --rpckey
 	args = append(args, fmt.Sprintf("--rpckey=%s", n.keyFile))
-	// --txindex
-	args = append(args, "--txindex")
```

Comment:
Might be a bit reckless to change the defaults. Could you gate the change behind a flag in the nodeConfig instead?